### PR TITLE
88560300 expire token 1

### PR DESCRIPTION
**Pull request by phirefly**
This test was accidentally added to master and didn't fail because there was an undefined step. This commit will expire tokens when approving through the web app instead of an email link.

### CODE REVIEW

File: app/controllers/communicarts_controller.rb [hunk 1]
```diff
@@ -21,8 +21,9 @@ def send_cart
   def approval_response
     cart = Cart.find(params[:cart_id]).decorate
     client_data = cart.proposal.client_data_legacy
-    approval = cart.approvals.find_by(user_id: user_id) 
-    
+    approval = cart.approvals.find_by(user_id: user_id)
+    @token ||= ApiToken.find_by(approval_id: approval.id)
```
Comment (afeld) [inline app/controllers/communicarts_controller.rb]: Why is this necessary?

**Comment by cmc333333** [comment]
Makes sense to me and patches up a small security hole
